test/__snapshots__/srcmak.test.ts.snap

```diff
@@ -2,108 +2,6 @@
 
 exports[`csharp + different entrypoint 1`] = `
 Object {
-  "Hello.World/.jsii": "{
```

Comment:
Any idea why this doesn't exist in the output anymore?

Reply:
I'm actually confused as to why it existed before (I know I added it) since the output jsii option has never been set on that test.
It being removed does seem like the correct state though.